mmv1/templates/terraform/examples/blockchain_nodes_basic.tf.erb

```diff
@@ -0,0 +1,3 @@
+resource "google_blockchain_node_engine_blockchain_nodes" "<%= ctx[:primary_resource_id] %>" {
+  name = "<%= ctx[:vars]['node_name'] %>"
```

Comment:
The test generated from this example is erroring, as the `name` field cannot be set:

```
        Error: Value for unconfigurable attribute
        
          with google_blockchain_node_engine_blockchain_nodes.default_node,
          on terraform_plugin_test.tf line 3, in resource "google_blockchain_node_engine_blockchain_nodes" "default_node":
           3:   name = "tf_test_blockchain_nameiuxsraql91"
        
        Can't configure a value for "name": its value will be decided automatically
        based on the result of applying this configuration.
```

Please remove `name` from this example and add the fields [mentioned in this message](https://github.com/GoogleCloudPlatform/magic-modules/pull/9395#issuecomment-1793042351)